src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2357,7 +2357,6 @@ internal void Run(bool inputLoopIsNested)
                 Dbg.Assert(ui != null, "Host.UI should return an instance.");
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-windows`
Tests module table restrictions.Verifies that Get-Command does not expose private module function under system lock down
```powershell
Cannot invoke method. Method invocation is supported only on core types in this language mode.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageModules.Tests.ps1: line 559
559:                 $GetCommandPrivateFnCmdInfo = Get-Command -Name "PrivateFn1" 2> $null

```
Tests module table restrictions.Verifies that Get-Command does not expose private function after explicitly importing nested module file under system lock down
```powershell
Cannot invoke method. Method invocation is supported only on core types in this language mode.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageModules.Tests.ps1: line 593
593:                 $ReImportPrivateFnCmdInfo = Get-Command -Name "PrivateFn1" 2> $null

```
Native streams behavior with PowerShell.Error stream.preserves error stream as is with Out-String
```powershell
Expected strings to be the same, but they were different.
Expected length: 33
Actual length:   51
Strings differ at index 4.
Expected: 'foo\n\nbar\n\nbazmiddlefoo\n\nbar\n\nbaz\n'
But was:  'foo\n \n\n \nb...'
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeStreams.Tests.ps1: line 52
52:             ($out | Out-String).Replace("`r", '') | Should -BeExactly "foo`n`nbar`n`nbazmiddlefoo`n`nbar`n`nbaz`n"
```
Native streams behavior with PowerShell.Error stream.does not get truncated or split when redirected
```powershell
Expected 1, but got 2.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeStreams.Tests.ps1: line 62
62:             $e.Count | Should -Be 1
```